static/skin/taskbar.js

```diff
@@ -5,26 +5,29 @@ function htmlDecode(input) {
 
 (function ($) {
     const root = $( `link[type='root']` ).attr("href");
+    const bookName = window.location.pathname.split(`${root}/`)[1].split('/')[0];
 
     $( "#kiwixsearchbox" ).autocomplete({
 
-        source: `${root}/suggest?content=${window.location.pathname.split(`${root}/`)[1].split('/')[0]}`,
+        source: `${root}/suggest?content=${bookName}`,
         dataType: "json",
         cache: false,
 
         response: function( event, ui ) {
-
               for(const item of ui.content) {
                   item.label = htmlDecode(item.label);
                   item.value = htmlDecode(item.value);
```

Comment:
Since `path` is HTML-encoded in `suggestion.json` (that is important for producing a valid JSON in case if `path` contains a double quote symbol), it must be `htmlDecode`-ed here, too.